libraries/classes/Controllers/Server/PrivilegesController.php

```diff
@@ -229,7 +229,7 @@ public function index(): void
                         ($username ?? ''),
                         ($hostname ?? ''),
                         ($tablename ?? ($routinename ?? '')),
-                        ($db_name ?? ''),
+                        (Util::unescapeMysqlWildcards($db_name) ?? ''),
```

Comment:
This would probably fail at some point because of a null value
The fix I pushed as 7c17889d780fe839bfe12c66b5a8fe01f68bf503 is:
```suggestion
                        (Util::unescapeMysqlWildcards($db_name ?? '')),
```